build.gradle.kts

```diff
@@ -100,6 +107,7 @@ dependencies {
     testImplementation("org.mockito.kotlin:mockito-kotlin:4.0.0")
 
     testImplementation("org.testcontainers:testcontainers:1.17.4")
+    testImplementation("org.testcontainers:mysql:1.17.5")
```

Comment:
Minor : It will be nice to keep all test containers dependency on same 1.17.5 version

Reply:
Done